src/main/webapp/scripts/map.js

```diff
@@ -26,14 +55,39 @@ function initMap() {
   map.addListener("click", e => {
     placeMarkerAndPanTo(e.latLng, map);
   });
+  /*Add Hard-Coded Markers */
+  addMarker(map);
+}
+
+/* Temporary Function to Hold Marker for Sky Garden */
+function addMarker(map) {
+  const skyGarden = {lat: 51.510881, lng: -0.083751}
+  const marker = new google.maps.Marker({
+    position: skyGarden,
+    map: map,
+    title: "Sky Garden"
+  })
+  marker.addListener("click", () => {
+    document.getElementById("rec-container").style.display = "block";
+    map.setZoom(16);
+    map.setCenter(marker.getPosition());
+  });
 }
 
 /* Place Marker Where Map is Clicked On & Show Popup*/
 function placeMarkerAndPanTo(latLng, map) {
+  const greyIcon = {
+    url: "/images/grey-marker.png", // url
+    scaledSize: new google.maps.Size(30, 40), // scaled size
+    origin: new google.maps.Point(0, 0), // origin
```

Comment:
this is the default value so you can safely omit

Reply:
Thanks!